packages/core/admin/admin/src/content-manager/hooks/useDocumentActions.ts

```diff
@@ -395,6 +405,39 @@ const useDocumentActions: UseDocumentActions = () => {
     [createDocument, formatAPIError, toggleNotification, trackUsage]
   );
 
+  const [autoCloneDocument] = useAutoCloneDocumentMutation();
+  const autoClone: IUseDocumentActs['autoClone'] = React.useCallback(
+    async ({ model, sourceId }) => {
+      try {
+        const res = await autoCloneDocument({
+          model,
+          sourceId,
+        });
+
+        if ('error' in res) {
+          toggleNotification({ type: 'warning', message: formatAPIError(res.error) });
+
+          return { error: res.error };
+        }
+
+        toggleNotification({
+          type: 'success',
+          message: { id: getTranslation('success.record.save') },
```

Comment:
Would a more descriptive success message be better for cloning?

Reply:
Good thinking!